WearTilesKotlin/gradle/libs.versions.toml

```diff
@@ -1,12 +1,12 @@
 [versions]
-android-gradle-plugin = "8.10.0"
+android-gradle-plugin = "8.10.1"
 compose = "1.8.2"
 compose-compiler = "1.5.15"
```

Comment:
Get the compose compiler through Kotlin 2?

https://kotlinlang.org/docs/compose-compiler-migration-guide.html